pkg/postgresql/utils.go

```diff
@@ -404,7 +404,7 @@ func getConfigFilePGParameters(ctx context.Context, connParams ConnParams) (comm
 
 func ParseBinaryVersion(v string) (int, int, error) {
 	// extact version (removing beta*, rc* etc...)
-	regex, err := regexp.Compile(`.* \(PostgreSQL\) ([0-9\.]+).*$`)
+	regex, err := regexp.Compile(`.* \(PostgreSQL\) ([0-9\.]+).*`)
```

Comment:
Yes, it is not useful because `.*` and the other reason is that when I ran the tests/debugged on a machine with self-compiled postgres I noticed some the regex failed to match when the output to stdout of the version was terminated with '\r\n'...